### PR TITLE
Update github actions to use py3.10/3.11 for Ubuntu 22.04 support

### PR DESCRIPTION
**Pull request by nzlosh**

### CODE REVIEW

File: .github/workflows/pack-build_and_test.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ on:
       python-version:
         required: false
         type: string
-        default: '["3.6","3.8","3.9"]'
+        default: '["3.8", "3.9", "3.10", "3.11"]'
```
Comment (amanda11) [inline .github/workflows/pack-build_and_test.yaml:17]: Line 42 will need updating which says what the job output will be

**Review by amanda11 — COMMENTED** [review]
Best to amend the comment that references 3.6 as well

**Comment by jpavlav** [comment]
@mamercad Can we get another 👍 on this? It looks like it should do the trick and it's blocking a PR downstream that uses this repository.

**Review by mamercad — APPROVED** [review]
🚢

**Review by cognifloyd — APPROVED** [review]

**Pull request merged**